README.md

```diff
@@ -1 +1,14 @@
-# hw01
\ No newline at end of file
+# hw01
+
```

Comment:
I didn't know that you could code in R with these arrows <> and slashes, so I learned something new thanks to you! And I agree with everything wanitchayap has said but refrained from repeating the comments. It was cool seeing your work and kudos to you on learning some R before college -- it's definitely fun and useful!